stdlib/hashtbl.mli

```diff
@@ -64,11 +64,12 @@ type (!'a, !'b) t
 
 val create : ?random: (* thwart tools/sync_stdlib_docs *) bool ->
              int -> ('a, 'b) t
-(** [Hashtbl.create n] creates a new, empty hash table, with
-   initial size [n].  For best results, [n] should be on the
-   order of the expected number of elements that will be in
-   the table.  The table grows as needed, so [n] is just an
-   initial guess.
+(** [Hashtbl.create n] creates a new, empty hash table, with suggested
+   initial size [n].  For best results, [n] should be on the order of
```

Comment:
I am not sure if `suggested` is the right term here: the table is created with a concrete size. Maybe
```
[Hashtbl.create n] creates a new, empty hash table, with initial size greater or equal to the suggested size [n].
```
?

Reply:
Makes sense, I've applied your suggestion.